ui/imports/shared/views/chat/CompactMessageView.qml

```diff
@@ -591,17 +589,18 @@ Item {
                 }
             }
 
-//            Loader {
-//                id: transactionBubbleLoader
-//                active: contentType === Constants.messageContentType.transactionType
-//                anchors.top: parent.top
-//                anchors.topMargin: active ? (chatName.visible ? 4 : 6) : 0
-//                sourceComponent: Component {
-//                    TransactionBubbleView {
-//                        store: rootStore
-//                    }
-//                }
-//            }
+            Loader {
+                id: transactionBubbleLoader
+                active: contentType === Constants.messageContentType.transactionType
+                anchors.top: parent.top
+                anchors.topMargin: active ? (chatName.visible ? 4 : 6) : 0
+                sourceComponent: Component {
+                    TransactionBubbleView {
+                        //store: rootStore
```

Comment:
Can this comment be removed?

Reply:
@jrainville actually, not yet, I left it as a reference to know what store the `TransactionBubbleView` component points to, and `TransactionBubbleView` is not refactored yet I think.